ursapictura/ThriveWell-Server
# Created interfaces and basics for Repository and Service layer files.

## Detailed Description
Created content for Service and Repository Interfaces. Fleshed out basic implementation of Repository and Service Layers.

## How to Test
1.  pull down "interfaces" branch to local machine
2. verify accuracy of interface connections and the implementation of the Service and Repository layers.


## Types of Changes
